### ISSUE TITLE
No such file or directory

### ISSUE THREAD

**Issue by seokchul-kwon** [nomeata/secp265k1-lookup-table#2]
Hi Breitner,

First of all, I sincerely thank you for sharing an interesting program with me.

I'd like to run the program you released. So you're out of his normal download.

I'm using Ubuntu Linux.

And make command, error has occurred.

That there is no "secp256k1.h". So when I looked at the "multiple-of-g.c" source program, I saw several include "XXXXX.h".
ex) #include "secp256k1.h"
#include "scalar_impl.h"
#include "num_impl.h"
#include "field_impl.h"
#include "group_impl.h"
#include "ecmult_impl.h"
#include "scratch_impl.h"

I can't find the all header file no matter how much I look for it.

Where can I get it? I'd appreciate it if you could teach me.

Thank you.

**Comment by nomeata** [comment]
Did you clone git submodules as well?

**Comment by seokchul-kwon** [comment]
Thank you for your reply.

1. command was run as follows 
    git clone --recursive https://github.com/nomeata/secp265k1-lookup-table
    make
2, I have installed bsort
3. install pip fastecdsa was run.

I would appreciate it if you could let me know if I missed anything or if there is a wrong command.
(ex) additional installations or methods are required.

**Comment by seokchul-kwon** [comment]
> Did you clone git submodules as well?

Thank you for your reply.

command was run as follows
git clone --recursive https://github.com/nomeata/secp265k1-lookup-table
make
2, I have installed bsort
3, pip3 install fastecdsa.

I would appreciate it if you could let me know if I missed anything or if there is a wrong command.
(ex) additional installations or methods are required.

**Comment by nomeata** [comment]
Hmm, it's been a while. I'm afraid you'll have to figure it out yourself.

**Comment by seokchul-kwon** [comment]
> Hmm, it's been a while. I'm afraid you'll have to figure it out yourself.

All right, I'll try again.

Did I do it in order? Just tell me that.

thank you.

**Comment by nomeata** [comment]
Yes, I think so. Maybe the secp256k1 library needs to be configured or built before, though.

**Comment by seokchul-kwon** [comment]
> Yes, I think so. Maybe the secp256k1 library needs to be configured or built before, though.

Thank you very much for your sincere reply. I'll figure it out. I think it can be solved simply.

**Comment by lilyanatia** [comment]
simply adding `-Isecp256k1/include` to the gcc command line in the Makefile fixes this.
